### PR TITLE
Rework README.md

### PR DESCRIPTION
**Pull request by al1ych**
this PR reworks readme according to the structure mentioned in one of the lectures. also available in the description of the corresponding issue (https://github.com/InnoQueue/Backend/issues/94)

### CODE REVIEW

**Comment by al1ych** [comment]
suspended

**Review by Dablup — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -131,9 +147,41 @@ where **firebase.json** is your json credentials file.
   Run `Deploy DEV` workflow.
 - `innoqueue` is deployd manually from main branch.
 
-> Features on which we are working can be found in [issues](https://github.com/InnoQueue/Backend/issues)
+## 📚 Contribution
+
+If you have any ideas on how to imporove the quality of our product or of the quality ensurance
```
Comment (SMore-Napi) [inline README.md:152]: If you have any **idea**

**Review by SMore-Napi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (al1ych) [reply README.md:152]: no

**Comment by al1ych** [comment]
todo:
- [ ] add styleguide.md as ref to readme.md

**Comment by al1ych** [comment]
todo

**Review by SMore-Napi — APPROVED** [review]

**Review by SMore-Napi — APPROVED** [review]

**Review by Dablup — APPROVED** [review]

**Review by MikhailMarch — APPROVED** [review]

**Pull request merged**